CONTRIBUTING.md

```diff
@@ -5,8 +5,8 @@
   - [Using GitPod](#using-gitpod)
   - [Using a Traditional Dev Environment](#using-a-traditional-dev-environment)
 - [Running the Application](#running-the-application)
-  - [Docker Mode](#running-the-application)
```

Comment:
The #docker-mode and #manual-mode don't work as in-page anchors since they are inside of a collapsible block.

It may be possible to rework the collapsible headings with HTML or something, but I messed with it a bit and having it jump to the parent section was the best quick fix I could come up with.

In short, leaving these as #running-the-application may be best for now.

Reply:
Ooops.  VSCode keeps 'fixing' that when I save. I'll revert it.